code/modules/spells/aoe_turf/conjure/doppelganger.dm

```diff
@@ -16,6 +16,17 @@
 
 var/list/doppelgangers = list()
 
+// Sanity : don't copy more than one guy
+/spell/aoe_turf/conjure/cast_check(skipcharge = 0,mob/user = usr)
+	var/list/L = view(user, 0)
+	var/list/mob/moblist = list()
+	L -= user
+	for (var/mob/M in L)
```

Comment:
Since we just need one mob in `moblist` to fail, I think of putting a `return FALSE` in the for loop.
What do you think ?

Reply:
Yes that's definitely a better approach.